nmmo/render/overlay.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+
+from nmmo.lib.colors import Neon
+from nmmo.systems import combat
+
+from .render_utils import normalize, make_two_tone
+
+# pylint: disable=unused-argument
+class OverlayRegistry:
+  def __init__(self, realm, renderer):
+    '''Manager class for overlays
+
+    Args:
+        config: A Config object
+        realm: An environment
+    '''
+    self.initialized = False
+
+    self.realm  = realm
+    self.config = realm.config
+    self.renderer = renderer
+
+    self.overlays = {
+       #'counts':     Counts, # TODO: change population to team
+       'skills':     Skills,
+       'wilderness': Wilderness}
```

Comment:
I vote for removing Wilderness if it's not used

Reply:
The Wilderness class is gone, but the packet still has `wilderness` to maintain the client compatibility.